lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -72,7 +71,7 @@ export function extractPackageFile(
   const endpoint = GlobalConfig.get('endpoint');
   try {
     // TODO: use schema (#9610)
-    const doc = parseSingleYaml<GitlabPipeline>(replaceReferenceTags(content));
+    const doc = parseSingleYaml<GitlabPipeline>(content);
```

Comment:
move to seperate PR, this is a refactor.

Reply:
See https://github.com/renovatebot/renovate/pull/31787